client/app/bundles/course/survey/containers/QuestionFormDialogue/QuestionForm.jsx

```diff
@@ -89,254 +98,387 @@ const questionFormTranslations = defineMessages({
+                name="min_options"
+                control={control}
+                render={({ field, fieldState }) => (
+                  <FormTextField
+                    field={field}
+                    fieldState={fieldState}
+                    disabled={disabled}
+                    fullWidth
+                    label={<FormattedMessage {...translations.minOptions} />}
+                    InputLabelProps={{
+                      shrink: true,
+                    }}
+                    onWheel={(event) => event.currentTarget.blur()}
+                    placeholder={
+                      questionFormTranslations.noRestriction.defaultMessage
+                    }
+                    renderIf={isMultipleResponse}
+                    style={styles.numberOfResponsesField}
+                    type="number"
+                    variant="standard"
+                  />
+                )}
+              />
+              <Controller
+                name="max_options"
+                control={control}
+                render={({ field, fieldState }) => (
+                  <FormTextField
+                    field={field}
+                    fieldState={fieldState}
+                    disabled={disabled}
+                    fullWidth
+                    label={<FormattedMessage {...translations.maxOptions} />}
+                    InputLabelProps={{
+                      shrink: true,
+                    }}
+                    onWheel={(event) => event.currentTarget.blur()}
+                    placeholder={
+                      questionFormTranslations.noRestriction.defaultMessage
+                    }
+                    renderIf={isMultipleResponse}
+                    style={styles.numberOfResponsesField}
+                    type="number"
+                    variant="standard"
+                  />
+                )}
+              />
+            </>
+          )}
+        </div>
+        <div>
+          {renderOptionsToDelete()}
+          {errors.options && (
+            <div style={{ color: red[500] }}>
+              <FormattedMessage {...errors.options.message} />
+            </div>
+          )}
+          <QuestionFormOptions
+            fieldsConfig={{
+              control,
+              fields: controlledOptionsFields,
+              append: optionsAppend,
+              remove: optionsRemove,
+            }}
+            deletedOptionsAppend={deletedOptionsAppend}
+            multipleChoice={isMultipleChoice}
+            multipleResponse={isMultipleResponse}
+          />
+        </div>
+      </div>
     );
-  }
-
-  render() {
-    const { handleSubmit, intl, onSubmit, disabled, formValues } = this.props;
-    const { TEXT, MULTIPLE_CHOICE, MULTIPLE_RESPONSE } = questionTypes;
-    const questionType = formValues && formValues.question_type;
+  };
 
-    return (
-      <Form onSubmit={handleSubmit(onSubmit)} encType="multipart/form-data">
-        <Field
+  return (
+    <>
```

Comment:
No need for this unnecessary wrapping in fragment. Just return `<form>`

Reply:
Got it!